### DISCUSSION TITLE
全局规则怎么设置排除app

### DISCUSSION THREAD

**Discussion by jiuqianyuan** [gkd-kit/gkd#418] [Q&A] [answered]
刚刚设置了一个全局规则，发现在某app有误触，在最新的描述里没找到怎么写app排除清单，求大佬解惑

https://github.com/gkd-kit/subscription/blob/f02719e3cad63a1af63e878405b9df25c9656bd8/src/types.ts#L263-L273

**Comment by lisonge** [comment, marked as answer]
https://github.com/gkd-kit/subscription/blob/f02719e3cad63a1af63e878405b9df25c9656bd8/src%2Ftypes.ts#L254-L262
